vowpalwabbit/exploration.cc

```diff
@@ -0,0 +1,121 @@
+#include "exploration.h"
+
+#include <stdexcept>
+#include <algorithm>
+#include <numeric>
+#include <cstring>
+
+using namespace std;
+
+namespace exploration
+{
+  void epsilon_greedy(float epsilon, uint32_t top_action, float* probability_distribution, uint32_t num_actions)
```

Comment:
We wanted to create a modestly more convenient C++ interface on top of the C interface like:
void epsilon_greedy(float epsilon, uint32_t top_action, vector<float> probability_distribution);
Can you add that?  (the above calling the C interface)

Reply:
done